package.json

```diff
@@ -25,19 +26,52 @@
   "homepage": "https://github.com/andela-oaboluwarin/checkpoint-1-inverted-index#readme",
   "devDependencies": {
     "browser-sync": "^2.18.6",
-    "eslint": "^3.13.1",
+    "browserify": "^13.3.0",
+    "eslint": "^3.14.0",
     "eslint-config-standard": "^6.2.1",
     "eslint-plugin-promise": "^3.4.0",
     "eslint-plugin-react": "^6.9.0",
     "eslint-plugin-standard": "^2.0.1",
     "gulp": "^3.9.1",
-    "gulp-changed": "^1.3.2",
-    "gulp-clean": "^0.3.2",
-    "gulp-imagemin": "^3.1.1"
+    "gulp-eslint": "^3.0.1",
+    "gulp-run": "^1.7.1",
+    "karma": "^1.4.0",
+    "karma-chrome-launcher": "^2.0.0",
+    "karma-coverage": "^1.1.1",
+    "karma-phantomjs-launcher": "^1.0.2",
+    "karma-spec-as-html-reporter": "0.0.3",
+    "karma-spec-reporter": "0.0.26",
+    "karma-verbose-reporter": "0.0.5",
+    "vinyl-source-stream": "^1.1.0",
+    "watchify": "^3.9.0"
   },
   "dependencies": {
+    "browser-sync": "^2.18.6",
+    "browserify": "^13.3.0",
+    "coveralls": "^2.11.15",
+    "eslint": "^3.14.0",
     "eslint-config-airbnb": "^14.0.0",
+    "eslint-config-standard": "^6.2.1",
     "eslint-plugin-import": "^2.2.0",
-    "eslint-plugin-jsx-a11y": "^3.0.2"
+    "eslint-plugin-jsx-a11y": "^3.0.2",
+    "eslint-plugin-promise": "^3.4.0",
+    "eslint-plugin-react": "^6.9.0",
+    "eslint-plugin-standard": "^2.0.1",
```

Comment:
I believe your eslint node packages should be in your dev-dependencies, based on your configured tasks, as you would not require them to start your application.